ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -282,16 +293,42 @@ def get_xunit_content(report, testname, elapsed):
 """ % data
 
     # output list of checked files
-    data = {
-        'escaped_files': escape(''.join(['\n* %s' % r
-                                         for r in sorted(report.keys())])),
-    }
-    xml += """  <system-out>Checked files:%(escaped_files)s</system-out>
+    if skip:
+        data = {
+            'skip': skip,
+        }
+        xml += """  <system-err>Tests Skipped due to %(skip)s</system-err>
+""" % data
+    else:
+        data = {
+            'escaped_files': escape(''.join(['\n* %s' % r
```

Comment:
Nitpick: same wrapping here.

Reply:
I was just indenting the existing code, but fixed anyway.